src/components/healthtips.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react';
+import { getInitalData } from '../store/reducer/user';
```

Comment:
Also there is no need of initialdata here. actually its the data of registered users from the backend and we don't need that here in this page IMO.

Reply:
Thankyou for letting me know the functionality of getInitialData. I have removed this from the file and  redux because it was being used for the same.